### DISCUSSION TITLE
The SLA report is not displayed correctly in PC version of DingTalk

### DISCUSSION THREAD

**Discussion by codering** [megaease/easeprobe#309] [Q&A]
The SLA report shows a problem on the PC version of DingTalk, but it is displayed normally on the mobile phone. Is this a bug in DingTalk?

PC :
<img width="355" alt="image" src="https://user-images.githubusercontent.com/2271825/224600529-bbb7e049-5412-4afe-87ce-8c53430618b4.png">

Mobile: 
![image](https://user-images.githubusercontent.com/2271825/224600541-aa04bec0-5041-4c9a-8050-225163a90fe7.png)

**Comment by haoel** [comment]
it looks like the bug of dingtalk.

**Reply by codering** [reply]
DingTalk does not support block syntax... 😂

see [支持的Markdown语法](https://open.dingtalk.com/document/orgapp/enterprise-internal-robots-send-markdown-messages#title-w87-omz-3es)

**Reply by haoel** [reply]
if you want to DingTalk can show it normally, we can ask DingTalk support the correct the Markdown, or we introduce the DingTalk format in EaseProbe to adapt it.

**Reply by codering** [reply]
I didn't find a good entrance to report to DingTalk. It would be more efficient if easeprobe did it.
